test/libdevcrypto/SecretStore.cpp

```diff
@@ -215,5 +216,122 @@ BOOST_AUTO_TEST_CASE(recode)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(vector_PBKDF2SHA256, *utf::expected_failures(1))
```

Comment:
What this `vector_` prefix in the test name suppose to mean? Name it `KeyImport` or something like that.